examples/_layouts/segmentation-tool.html

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+---
+
+<section class="section">
+  <div class="container">
+    <h1 class="title is-1">{{ page.title }}</h1>
+
+    {% include customButtons/modeButtons.html %} {% if page.customButtons %} {%
```

Comment:
You can wrap these in prettier/eslint ignores if it's killing your formatting.

Reply:
HTML

```html
<!-- prettier-ignore-start -->

<!-- prettier-ignore-end -->
```

Reply:
Note to self again: Lets talk examples.